definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -6726,7 +6684,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "boAddTranslations",
     "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "CRU"
```

Comment:
Just need to keep this original object in "AuthorisationCaseEvent-prod.json".

Ideally, **if** Jeremy confirms every role should have "R" access. it would be to have the following changes:
inside prod file:
- original authorisations
inside nonprod file:
- caseworker-divorce-solicitor role with R acess

Reply:
Updated: 67fd1ae81a33cefecd64d189c0c637ad893761be